patches/minecraft/net/minecraft/item/ItemBow.java.patch

```diff
@@ -21,11 +21,47 @@
                  if ((double)f >= 0.1D)
                  {
 -                    boolean flag1 = flag && itemstack.func_77973_b() == Items.field_151032_g;
-+                    boolean flag1 = flag && itemstack.func_77973_b() instanceof ItemArrow; //Forge: Fix consuming custom arrows.
++                	boolean isArrowInfinite = determineIsArrowInfinite(flag, itemstack);
```

Comment:
https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge#tips

> Space instead of tabs